pkgs/applications/graphics/digikam/default.nix

```diff
@@ -55,14 +55,14 @@
 
 mkDerivation rec {
   pname   = "digikam";
-  version = "7.3.0";
+  version = "7.4.0";
 
   src = fetchurl {
-    url = "mirror://kde/stable/${pname}/${version}/${pname}-${version}.tar.xz";
-    sha256 = "sha256-la6pO+HP05u1IzO4Kz5Xv2gIDH0TGddU0WeiD22+RVE=";
+    url = "mirror://kde/stable/${pname}/${version}/digiKam-${version}.tar.xz";
+    sha256 = "sha256-0Iq2bacyu0SbwQEG7BHdne+ls1Yt7TdBsEHbuqcVUEo=";
   };
 
-  nativeBuildInputs = [ cmake doxygen extra-cmake-modules kdoctools wrapGAppsHook ];
+  nativeBuildInputs = [ cmake doxygen extra-cmake-modules kdoctools wrapGAppsHook wrapQtAppsHook ];
```

Comment:
The [manual states](https://nixos.org/manual/nixpkgs/unstable/#qt-default-nix-co-2): `All Qt packages must include wrapQtAppsHook in nativeBuildInputs, or you must explicitly set dontWrapQtApps`.

Reply:
That said it seems to function fine without this, so I'm kind of posing this as a question actually.

Reply:
Seems that the app has been wrapped already, so should be fine not to add this.

Reply:
Went ahead and reverted this part.